src/router.ts

```diff
@@ -0,0 +1,43 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import Home from './components/Home.vue'
+import User from './components/User.vue'
+import UserDetail from './components/UserDetail.vue'
+import Project from './components/Project.vue'
+import Role from './components/Role.vue'
+import Search from './components/Search.vue'
+
+const routes = [
+  {
+    path: '/',
+    component: Home
+  },
+  {
+    path: '/user',
+    component: User,
+    name: 'User'
```

Comment:
di router ini ada yg pake name: value, trus ada jg yg ngga, name ini berarti buat apa ya?

Reply:
an alternative way to call the path
e.g. <router-link :to="{name: User ...}" or <router-link :to="/user" for a simple way

mostly name use when we need complex routing or only to give identifier about the path